Bonsai/Bonsai.csproj

```diff
@@ -40,45 +38,84 @@
+    <Error Condition="!Exists('$(OutputPath)Bonsai.exe')" Text="Project must be built before repacking. '$(OutputPath)Bonsai.exe' not found." />
+    <Error Condition="'$(RepackedOutputPath)' == '' or '$(RepackedOutputFilePath)' == ''" Text="Repacked output path not configured." />
+
+    <MakeDir Directories="$(RepackedOutputPath)" />
+
     <PropertyGroup>
-      <WorkingDirectory>$(MSBuildThisFileDirectory)bin\$(Configuration)\$(TargetFramework)</WorkingDirectory>
+      <_IlRepackCommand>
+"$(ILRepack)"
+/verbose
+/out:"$(RepackedOutputFilePath)"
+/internalize:"$(MSBuildThisFileDirectory)InternalizeExcludePatterns"
+"$(OutputPath)Bonsai.exe"
+"$(OutputPath)System.Collections.Immutable.dll"
+"$(OutputPath)System.Reflection.Metadata.dll"
+"$(OutputPath)System.Reflection.MetadataLoadContext.dll"
+"$(OutputPath)Bonsai.Configuration.dll"
+"$(OutputPath)Bonsai.NuGet.dll"
+"$(OutputPath)Bonsai.NuGet.Design.dll"
+"$(OutputPath)Newtonsoft.Json.dll"
+"$(OutputPath)NuGet.Common.dll"
+"$(OutputPath)NuGet.Configuration.dll"
+"$(OutputPath)NuGet.Frameworks.dll"
+"$(OutputPath)NuGet.Packaging.dll"
+"$(OutputPath)NuGet.Protocol.dll"
+"$(OutputPath)NuGet.Resolver.dll"
+"$(OutputPath)NuGet.Versioning.dll"
+      </_IlRepackCommand>
     </PropertyGroup>
+    <Exec WorkingDirectory="$(RepackedOutputPath)" Command="$(_IlRepackCommand.Replace('%0D', ' ').Replace('%0A', ' '))" UseCommandProcessor="false" />
+  </Target>
+
+  <Target Name="UseRepackOutputForPackage" Condition="'$(UseRepackForBootstrapperPackage)' == 'true'" AfterTargets="BuiltProjectOutputGroup" DependsOnTargets="ConfigureRepack">
+    <Error Condition="!Exists('$(RepackedOutputFilePath)')" Text="Project must be repacked before repacking. '$(RepackedOutputFilePath)' not found." />
     <ItemGroup>
-      <InputAssemblies Include="System.Buffers.dll" />
-      <InputAssemblies Include="System.Memory.dll" />
-      <InputAssemblies Include="System.Numerics.Vectors.dll" />
-      <InputAssemblies Include="System.Collections.Immutable.dll" />
-      <InputAssemblies Include="System.Reflection.Metadata.dll" />
-      <InputAssemblies Include="System.Reflection.MetadataLoadContext.dll" />
-      <InputAssemblies Include="System.Runtime.CompilerServices.Unsafe.dll" />
-      <InputAssemblies Include="Bonsai.Configuration.dll" />
-      <InputAssemblies Include="Bonsai.NuGet.dll" />
-      <InputAssemblies Include="Bonsai.NuGet.Design.dll" />
-      <InputAssemblies Include="Newtonsoft.Json.dll" />
-      <InputAssemblies Include="NuGet.Common.dll" />
-      <InputAssemblies Include="NuGet.Configuration.dll" />
-      <InputAssemblies Include="NuGet.Frameworks.dll" />
-      <InputAssemblies Include="NuGet.Packaging.dll" />
-      <InputAssemblies Include="NuGet.Protocol.dll" />
-      <InputAssemblies Include="NuGet.Resolver.dll" />
-      <InputAssemblies Include="NuGet.Versioning.dll" />
+      <BuiltProjectOutputGroupOutput Remove="@(BuiltProjectOutputGroupOutput)" />
```

Comment:
That is a mouthful, can we rename one of the `Output` words with something else like `Files` or `Artifacts`?

Reply:
As noted during our meeting, I didn't name these. The .NET SDK devs did 😅